modules/aruco/src/aruco.cpp

```diff
@@ -1278,7 +1278,7 @@ void GridBoard::draw(Size outSize, OutputArray _img, int marginSize, int borderB
 Ptr<Board> Board::create(InputArrayOfArrays objPoints, const Ptr<Dictionary> &dictionary, InputArray ids) {
 
     CV_Assert(objPoints.total() == ids.total());
-    CV_Assert(objPoints.type() == CV_32FC3);
+    CV_Assert(objPoints.type() == CV_32F);
```

Comment:
We can support both formats, so here can be `CV_Assert(objPoints.type() == CV_32FC3 || objPoints.type() == CV_32FC1);`